neo/Consensus/ConsensusContext.cs

```diff
@@ -213,6 +215,15 @@ public ConsensusPayload MakePrepareRequest()
             IEnumerable<Transaction> memoryPoolTransactions = Blockchain.Singleton.MemPool.GetSortedVerifiedTransactions();
             foreach (IPolicyPlugin plugin in Plugin.Policies)
                 memoryPoolTransactions = plugin.FilterForBlock(memoryPoolTransactions);
+
+            if (nextBlockLight)
```

Comment:
Maybe we can have some functions for leaving the code more clear.
If @erikzhang and @igormcoelho agree with this change I can help to try to organize it better.